test/security/engines/KeyStore.spec.js

```diff
@@ -0,0 +1,199 @@
+const chai = require('chai');
```

Comment:
this was in its own dir, test/security/keystores/KeyStore.spec.js . I wasn't sure whether to mv it to test/security/engines or mv the KeyStore classes to src/security/keystores . I flipped a coin and it said mv this to test/security/engines. But I could be persuaded to not-follow the coin's advice; the coin isn't always right.